### PR TITLE
Switch to github actions

### PR DESCRIPTION
**Pull request by sarahet**

### CODE REVIEW

**Comment by sarahet** [comment]
This works now for MacOS and Linux. Should we delete the travis yml now?

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -24,11 +24,19 @@ endforeach()
...
-            foreach(FF m0 m8 m9 sam bam m9.gz sam.bz2)
-                add_test (NAME test_search_${PROG}_${DI}_${MODE}_${FF}
-                        COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/maintests.sh
-                    "${CMAKE_SOURCE_DIR}" "${CMAKE_BINARY_DIR}" ${PROG} ${DI} ${MODE} "SEARCH" ${FF})
-            endforeach()
+            if(CMAKE_SYSTEM_NAME STREQUAL Linux)
+                foreach(FF m0 m8 m9 sam bam m9.gz sam.bz2)
+                    add_test (NAME test_search_${PROG}_${DI}_${MODE}_${FF}
+                            COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/maintests.sh
+                        "${CMAKE_SOURCE_DIR}" "${CMAKE_BINARY_DIR}" ${PROG} ${DI} ${MODE} "SEARCH" ${FF})
```
Comment (h-2) [inline tests/CMakeLists.txt:31]:
```suggestion
                    add_test (NAME test_search_${PROG}_${DI}_${MODE}_${FF}
                              COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/maintests.sh
                              "${CMAKE_SOURCE_DIR}" "${CMAKE_BINARY_DIR}" ${PROG} ${DI} ${MODE} "SEARCH" ${FF})
```

💅🏻

File: tests/CMakeLists.txt [hunk 2]
```diff
@@ -24,11 +24,19 @@ endforeach()
 foreach(PROG ${PROGS})
     foreach(DI fm bifm)
         foreach(MODE ${MODES})
-            foreach(FF m0 m8 m9 sam bam m9.gz sam.bz2)
-                add_test (NAME test_search_${PROG}_${DI}_${MODE}_${FF}
-                        COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/maintests.sh
-                    "${CMAKE_SOURCE_DIR}" "${CMAKE_BINARY_DIR}" ${PROG} ${DI} ${MODE} "SEARCH" ${FF})
-            endforeach()
+            if(CMAKE_SYSTEM_NAME STREQUAL Linux)
```
Comment (h-2) [inline tests/CMakeLists.txt:27]: Can you replace this with "not macOS" for now since the tests do work on FreeBSD?

**Comment by h-2** [comment]
> This works now for MacOS and Linux. Should we delete the travis yml now?

Yep!

File: tests/CMakeLists.txt (hunk 2 above)
Comment (sarahet) [reply tests/CMakeLists.txt:27]: Yes no problem

**Review by h-2 — APPROVED** [review]

**Pull request merged**